src/main/java/com/puppycrawl/tools/checkstyle/api/TokenTypes.java

```diff
@@ -195,37 +195,29 @@
      * <p>parses as:</p>
      *
      * <pre>
-     * +--METHOD_DEF
-     *     |
-     *     +--MODIFIERS
-     *         |
-     *         +--LITERAL_PUBLIC (public)
-     *         +--LITERAL_STATIC (static)
-     *     +--TYPE
-     *         |
-     *         +--LITERAL_INT (int)
-     *     +--IDENT (square)
-     *     +--PARAMETERS
-     *         |
-     *         +--PARAMETER_DEF
-     *             |
-     *             +--MODIFIERS
-     *             +--TYPE
-     *                 |
-     *                 +--LITERAL_INT (int)
-     *             +--IDENT (x)
-     *     +--SLIST ({)
-     *         |
-     *         +--LITERAL_RETURN (return)
-     *             |
-     *             +--EXPR
-     *                 |
-     *                 +--STAR (*)
-     *                     |
-     *                     +--IDENT (x)
-     *                     +--IDENT (x)
-     *             +--SEMI (;)
-     *         +--RCURLY (})
+     * |--METHOD_DEF -&gt; METHOD_DEF
```

Comment:
remove `|` it is not required as it not lead to anywhere.

Reply:
Done.